test/runtests.jl

```diff
@@ -4,6 +4,31 @@ using Random
 using Test
 
 @testset "ChainRulesTestUtils.jl" begin
+    include("to_vec.jl")
+
+    @testset "isapprox" begin
+        @testset "Composite{Tuple}" begin
+            @testset "basic" begin
+                x_tup = (1.5, 2.5, 3.5)
+                x_comp = Composite{typeof(x_tup)}(x_tup...)
+                @test x_comp ≈ x_tup
+                @test x_tup ≈ x_comp
+                @test x_comp ≈ x_comp
+
+                @test_throws Exception x_comp ≈  collect(x_tup)
+            end
+
+            @testset "different types" begin
+                # both of these are reasonable diffentials for the `Tuple{Int, Int}` primal
+                @test Composite{Tuple{Int, Int}}(1f0, 2f0) ≈ Composite{Tuple{Int, Int}}(1.0, 2.0)
+
+                # But these have different primals so should not be equal
```

Comment:
It's not clear to me how this comment ties up with the code below. If they're not meant to be equal, then why are they approximately equal?

Reply:
oh dear that test should say `!≈`  something is wrong there